docs/docs/installation/installing-rasa-open-source.mdx

```diff
@@ -89,6 +84,25 @@ to install all needed dependencies for every configuration.
 
 :::
 
+*If you are using macOS*, note that installing `rasa[full]` (either via pip or from source) could result in a failure
```

Comment:
I couldn't find anywhere in the docs where we specify supported Python versions 🤔

Reply:
I remember a discussion about it not being clearly explained in the docs

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/11773/commits/bfd5135e0d7127da726a92b50602c29ae1991636